RecoLocalCalo/HcalRecProducers/python/HBHEMethod2Parameters_cfi.py

```diff
@@ -6,9 +6,9 @@
     applyPedConstraint    = cms.bool(True),
     applyTimeConstraint   = cms.bool(True),
     applyPulseJitter      = cms.bool(False),
-    applyTimeSlew         = cms.bool(True),           #units
-    ts4Min                = cms.double(0.),           #fC
-    ts4Max                = cms.vdouble(100.,45000.), #fC # this is roughly 20 GeV
+    applyTimeSlew         = cms.bool(True),       #units
+    ts4Min                = cms.double(0.),       #GeV, for run2-2016 0 fC used
+    ts4Max                = cms.vdouble(20.,20.), #GeV, for run2-2016 100 fC used
```

Comment:
similar to another comment alluding to older historical value, the ```for run2-2016 X fC used``` is rather confusing.
If you want to keep this message, please be more verbose, maybe refer to some twiki or other document which has documentation for all the variables and their evolution across releases.